sd-core/src/hypergraph/traits.rs

```diff
@@ -17,6 +17,7 @@ pub enum WireType {
     Data,
     ControlFlow,
     SymName,
+    Colour(ecolor::Color32),
```

Comment:
Do we use the other wire types anywhere else? Can we just replace this whole enum by a colour?

Reply:
I suppose we could but it's less semantic. There is currently a bunch of custom logic for both control flow and sym name linking and it's not trivial to get rid of that also